projects/pastanaga/src/lib/controls/control.model.ts

```diff
@@ -1,4 +1,4 @@
-export interface IControlModel {
+export interface ControlModelData {
```

Comment:
Related with the conversation we had about this I thought that the `IControlModel` objects were coming from backend that was why I didn't like to have class instances inside its children. Thanks for the help!